dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -475,20 +530,16 @@ public void ContentFallbackFalse_PageFallbackTrue_PageSpanish_ViewEnglish404_Vie
      *
      */
     @Test (expected = HTMLPageAssetNotFoundException.class)
-    public void ContentFallbackFalse_PageFallbackFalse_PageEnglish_ViewEnglishContent1And2_ViewSpanish404() throws Exception{
+    @UseDataProvider("cases")
+    public void ContentFallbackFalse_PageFallbackFalse_PageEnglish_ViewEnglishContent1And2_ViewSpanish404(final Container container, final Template template) throws Exception{
```

Comment:
I would add javadoc explaining better the given scenario and expected result

Reply:
this test is not new